airflow/models/__init__.py

```diff
@@ -69,6 +69,7 @@ def import_all_models():
     import airflow.models.dataset
     import airflow.models.serialized_dag
     import airflow.models.tasklog
+    import airflow.www.fab_security.sqla.models
```

Comment:
(note to self: I don't think this is needed, check it)

Reply:
Yep, already done here:
https://github.com/apache/airflow/blob/fb7c6afc8cb7f93909bd2e654ea185eb6abcc1ea/airflow/utils/db.py#L667

Reply:
It fails without that pretty sure

Reply:
What's "it"?

Reply:
Nevermind, I see what's going on now. This can probably be cleaned up, but I'm not going to pull that string right now. I've put this back.

Reply:
Should we import the TaskNote and DagrunNote tables in this place instead?

Reply:
Okay, I've pulled this out into #27869, just to keep things a little cleaner.